src/print.jl

```diff
@@ -549,6 +549,14 @@ function function_string(print_mode, constraint::AbstractConstraint)
     return function_string(print_mode, f)
 end
 
+"""
+    in_set_string(print_mode::Type{<:PrintMode}, set)
+
+Return a `String` representing the membership to the set `set` using print mode
+`print_mode`.
+"""
+function in_set_string end
```

Comment:
@blegat, there is not a generic docstring.

Reply:
Yes but the other case is not covered since the second argument is a constraint, not a set.